tests/unit_tests/gui/test_csv_export.py

```diff
@@ -53,9 +51,13 @@ def handle_finished_box():
     QTimer.singleShot(3000, handle_finished_box)
 
     gui.tools["Export data"].trigger()
-    assert file_name == "output.csv"
+    assert file_name == export_path
```

Comment:
I agree to have it as a parameter; ie. `export_path` but I'd inline into function body instead (maybe after `file_name = None`) since it's not a fixture nor parametrize object.

Reply:
Done, thanks.